examples/simple-server.rs

```diff
@@ -52,7 +52,7 @@ use std::{num::NonZeroUsize, str::FromStr, time::Duration};
 use url::Url;
 use vbs::version::StaticVersionType;
 
-const NUM_NODES: usize = 2;
+const NUM_NODES: usize = 5;
```

Comment:
Why?

Reply:
some tests were stuck in View sync so this was an attempt to fix those. I think it did fix it or maybe they are flaky